lib/puppet/provider/ios_radius_global/cisco_ios.rb

```diff
@@ -19,6 +19,7 @@ def self.instances_from_cli(output)
     new_instance[:attributes] = PuppetX::CiscoIOS::Utility.parse_multiples(output, commands_hash, 'attributes', 0) || []
     new_instance = new_instance.merge(Puppet::Provider::RadiusGlobal::CiscoIos.instances_from_cli(output).first)
     new_instance.delete_if { |_k, v| v.nil? }
+    new_instance = Puppet::Provider::IosRadiusGlobal::CiscoIos.trim_attributes(new_instance)
```

Comment:
The `strip` approach is one way, or a rejig of the `get_value` in the command.yaml could be done to avoid having these work arounds in the provider.

`get_value: '.*(?:(?:radius-server attribute (?!list)(?<attribute_id>\S*)) (?<attribute_string>.*[^\s\\n]))'`

Should achieve the same thing - or alternatively it would be better to put the `strip` change in to the `parse_multiples` utility helper as this would cover a lot more potential areas where the regex was incorrect and picked up spurious spaces.

Reply:
Your regex fix worked